src/Microsoft.AspNet.OData.Shared/Extensions/ODataPathExtensions.cs

```diff
@@ -11,8 +11,16 @@
 
 namespace Microsoft.AspNet.OData.Extensions
 {
-    internal static class ODataPathExtensions
+    /// <summary>
+    /// Extensions method for <see cref="ODataPath"/>.
+    /// </summary>
+    public static class ODataPathExtensions
```

Comment:
Do we have tests for these extensions methods?

Reply:
I have added them